test/common/README.md

```diff
@@ -116,6 +116,14 @@ Checks whether `IPv6` is supported on this platform.
 
 Checks if there are multiple localhosts available.
 
+### hijackStderr/Stdout(listener)
```

Comment:
@jasnell I suggested the the merge (since it's an internal guide) https://github.com/nodejs/node/pull/13439#discussion_r120014579
Please 👍 / 👎 again, just so we don't ask @XadillaX do unnecessary work